locales/uk.json

```diff
@@ -126,7 +126,9 @@
 			"403": "Вибачте, але ми справді не можемо вам цього показати!",
 			"404": "Ой, цього сайту не існує.",
 			"dataProtection": "Політика конфіденційності",
+			"dataProtectionThr": "Політика конфіденційності",
 			"dataProtectionFile": "Privacy Policy of School",
+			"dataProtectionFileThr": "Privacy Policy of School",
```

Comment:
please fix that language typo as well as the one in line 130 above as you are here once ^^

Reply:
done